scripts/auto-rebase/rebase.sh

```diff
@@ -1226,10 +1229,37 @@ rebase_lvms_to() {
     rm -rf "${STAGING_DIR}"
 }
 
+to_just_images() {
+    local release_image_amd64=$1
+    local release_image_arm64=$2
+
+    download_release "${release_image_amd64}" "${release_image_arm64}"
+    checkout_rebase_branch
+    update_images
+
+    if [[ -n "$(git status -s assets/release packaging/crio.conf.d)" ]]; then
+        local last_rebase_script="${REPOROOT}/scripts/auto-rebase/last_rebase.sh"
+        rm -f "${last_rebase_script}"
+        cat - >"${last_rebase_script}" <<EOF
+    #!/bin/bash -x
+    ./scripts/auto-rebase/rebase.sh images-to "${release_image_amd64}" "${release_image_arm64}"
+EOF
+        chmod +x "${last_rebase_script}"
+
+        title "## Committing new image references"
+        git add assets/release
```

Comment:
Need to `git add` the last_rebase.sh script here, too.

Reply:
Done...